### PR TITLE
feat(invitation): add template for circle invitation

### PR DESCRIPTION
**Pull request by devformatters**

### CODE REVIEW

File: lang/zh-CN/circleInvitation.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title: 'Matters | {{sender.displayName}}正在邀请你进入{{name}}围炉，并为你开通了免费资格的限时福利'
+preview: '{{sender.displayName}} 正在邀请你加入{{name}}围炉，并为你开通了为期 {{freePeriod}} 个月的限时体验福利，你可以在这段时间内，免费体验围炉的所有功能！'
```
Comment (robertu7) [inline lang/zh-CN/circleInvitation.yaml:2]: `circle.displayName`?

File: lang/zh-CN/circleInvitation.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+content1: '{{sender.displayName}} 正在邀请你加入{{name}}围炉，并为你开通了为期 {{freePeriod}} 个月的限时体验福利，你可以在这段时间内，免费体验围炉的所有功能！'
```
Comment (robertu7) [inline lang/zh-CN/circleInvitation.yaml:4]: display names of user and circle should be hyperlinked?

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: lang/zh-CN/circleInvitation.yaml (hunk 2 above)
Comment (devformatters) [reply lang/zh-CN/circleInvitation.yaml]: Added.

File: lang/zh-CN/circleInvitation.yaml (hunk 1 above)
Comment (devformatters) [reply lang/zh-CN/circleInvitation.yaml]: Naming changed.

**Review by robertu7 — APPROVED** [review]
Let's try

**Pull request merged**